packages/mux-video/package.json

```diff
@@ -62,7 +62,8 @@
   "dependencies": {
     "@mux/playback-core": "0.18.5",
     "castable-video": "0.5.2",
-    "custom-media-element": "0.2.0"
+    "custom-media-element": "0.2.0",
+    "media-tracks": ">=0.2.1"
```

Comment:
praise: Good call/attention to detail on the semver qualifier here (re: playback-core version consistency) 🎉